### PR TITLE
Long pollling fallback

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to aspnet/announcements#470.

[Preview link](https://review.docs.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback?branch=pr-en-us-25621).

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

You can :shipit: when you're ready.

File: docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Breaking change: Blazor server: Disable LongPolling fallback transport"
+description: "Learn about the breaking change in ASP.NET Core 6.0 where LongPolling is not used as a fallback transport utilized when WebSockets aren't available."
```
Comment (TanayParikh) [inline docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md:3]:
```suggestion
description: "Learn about the breaking change in ASP.NET Core 6.0 where LongPolling is not used as a fallback transport when WebSockets aren't available."
```

File: docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ms.date: 08/12/2021
+no-loc: [ Blazor, WebSocket, LongPolling ]
+---
+# Blazor server: Disable long polling fallback transport
+
+*LongPolling* is a fallback transport utilized when WebSockets aren't available. The LongPolling fallback can lead to a degraded user experience, so it has been removed. Both the client and server now support only WebSockets by default.
```
Comment (TanayParikh) [inline docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md:9]:
```suggestion
Prior to .NET 6, *LongPolling* was a fallback transport utilized when WebSockets weren't available. The LongPolling fallback can lead to a degraded user experience, so it has been removed. Both the client and server now support only WebSockets by default.
```

File: docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+| Client | Server | Message |
+|---|---|---|
+| WS (without browser WS support) | WebSockets | `Unable to connect, please ensure you are using an updated browser that supports WebSockets.` |
+| WS (with WS connection being rejected) | WebSockets | `Unable to connect, please ensure WebSockets are available. A VPN or proxy may be blocking the connection.` |
+| WS | LongPolling | `An unhandled error has occurred.` Console Error: `Unable to initiate a SignalR connection to the server. This might be because the server is not configured to support WebSockets. To troubleshoot this, visit https://aka.ms/blazor-server-websockets-error.` |
+| LongPolling | WebSockets | `An unhandled error has occurred.` |
+
+## Reason for change
+
+This change was mode to improve the overall end-user experience by enforcing WebSocket use.
```
Comment (TanayParikh) [inline docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md:32]:
```suggestion
This change was made to improve the overall end-user experience by enforcing WebSocket use.
```

**Review by TanayParikh — APPROVED** [review]

File: docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md (hunk 3 above)
Comment (gewarren) [reply docs/core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md:32]: Good catches!

**Pull request merged**